addons/l10n_vn_edi_viettel/models/account_move.py

```diff
@@ -143,6 +143,16 @@ class AccountMove(models.Model):
         check_company=True,
         export_string_translation=False,
     )
+    l10n_vn_edi_reversed_entry_invoice_number = fields.Char(
+        string='Reversed Entry SInvoice Number',
+        related='reversed_entry_id.l10n_vn_edi_invoice_number',
+        help="Technical field for ui/ux"
+    )
+    l10n_vn_edi_replacement_origin_invoice_number = fields.Char(
+        string='Replacement Origin SInvoice Number',
+        related='l10n_vn_edi_replacement_origin_id.l10n_vn_edi_invoice_number',
+        help="Technical field for ui/ux"
+    )
```

Comment:
```suggestion
    l10n_vn_edi_reversed_entry_invoice_number = fields.Char(
        related='reversed_entry_id.l10n_vn_edi_invoice_number',
    )
```
The string/help are not needed. we don't plan to display these, so we may as well avoid adding strings to translate.

Reply:
Ok no need helpstring got it, done